src/GitHub.Api/Git/IRepository.cs

```diff
@@ -21,6 +21,7 @@ public interface IRepository : IEquatable<IRepository>
         ITask ListLocks();
         ITask RequestLock(string file);
         ITask ReleaseLock(string file, bool force);
+		ITask CheckoutFiles( List<string> files );
```

Comment:
This should be indented with spaces. There should be no spaces after `(` and before `)`